extensions/extended-blocks/core-embed/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+
+/**
+ * Re-activates the Facebook and Instagram embed block variations by
+ * making them appear in the inserter and their URL patterns parseable again.
+ *
+ * Relevant for Gutenberg >= 9.0, where the aforementioned blocks were deprecated*[0] because
+ * they do not support the oEmbed changes that Facebook is going to implement on Oct. 24th, 2020,
+ * and core has currently no plans to support it.
+ *
+ * However, we do plan on keeping support for them in Jetpack and WordPress.com by sending an
+ * access token alongside the oEmbed API request.
+ *
+ * Our goal is for this go unnoticed by our end-users, as this is only an implementation detail.
+ *
+ * *[0] https://github.com/WordPress/gutenberg/pull/24472.
+ *
+ * @param {object} settings - Block settings object.
+ * @param {string} name - The block name
+ * @returns {object} The settings for the given block with the patched variations.
+ */
+function reactivateFacebookAndInstagramEmbedBlockVariations( settings, name ) {
+	if ( name !== 'core/embed' || ! settings.variations ) {
```

Comment:
Do we need to account for regular WordPress 5.4 and 5.5 installations, where [this change](https://github.com/WordPress/gutenberg/pull/24090) hasn't happened yet and embed blocks are a bit different (`core-embed/facebook` I think?)?

Reply:
I tried to think this through, and I _think_ we don't have to do anything about them: Basically, they'll continue to exist in those versions, so unlike GB >= 9.0, nothing is being removed that we have to add back.

Does that make sense? :thinking:

Reply:
It does, you're right! 👍